### ISSUE TITLE
'ValidateAsync' in type 'AuthorizationValidationRule' does not have an implementation

### ISSUE THREAD

**Issue by RyanGhd** [graphql-dotnet/authorization#69]
`GraphQL.Authorization.AuthorizationValidationRule` throws the following exception:

```
Method 'ValidateAsync' in type 'GraphQL.Authorization.AuthorizationValidationRule' from assembly 'GraphQL.Authorization, Version=3.0.38.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```

Libs:
GraphQL 3.0.0-preview-1352
GraphQL.Authorization 3.0.38-preview-38

I think the reason is that it tries to implement IValidationRule which comes as follows:

```
namespace GraphQL.Validation
{
  public interface IValidationRule
  {
    Task<INodeVisitor> ValidateAsync(ValidationContext context);
  }
}
```

**Comment by sungam3r** [comment]
You are using a newer version of the graphql-dotnet package. Now you can roll back the version to that state of the code when method was still synchronous or wait for the new version of this package

**Comment by RyanGhd** [comment]
@sungam3r, 
grapql-dotnet repo recommends that: "The latest stable version 2.4.0 has many known issues that have been fixed in 3.0.0-preview-XXXX versions."
So I prefer to keep the new version.
 
what is the ETA for the new version that addresses this issue?

**Comment by sungam3r** [comment]
You **may** keep preview version. I am just talking about not the latest preview since it has conflict with authorization project. 

> what is the ETA for the new version that addresses this issue?

Although I am a member of `graphql-dotnet` org, I don’t have write permission in this repository. So now we are waiting for @joemcbride to merge #70.
